mkdocs.yml

```diff
@@ -12,7 +12,9 @@ theme:
     repo: fontawesome/brands/github
   features:
   - navigation.footer
-  - content.code.annotations # (1)!
+  - content.code.annotations
```

Comment:
Adding some things we added in jujuba to improve the UX around the mkdocs-material page